.github/workflows/documentation.yml

```diff
@@ -0,0 +1,35 @@
+name: DOCUMENTATION
+
+on: [push]
+
+jobs:
+
+  documentation:
+    runs-on: espressopp/buildenv:ubuntu
+
+    steps:
+    - name: build documentation
+      run: |
+        git clone https://github.com/espressopp/espressopp.git
+        mkdir espressopp-build
+        cd espressopp-build
+        cmake ../espressopp
+        make doc
+
+    - name: update docu repo
+      run: |
+        git clone --depth 1 https://github.com/espressopp/espressopp.github.io.git html;
+        git rm -rf html/latest
+        mv espressopp-build/doc/dg/html html/latest/doxygen;
+        mv espressopp-build/doc/ug/_build/latex/ESPResSo++.pdf html/latest/;
+        mv espressopp-build/doc/ug/_build/html/* html/latest/;
+        git config --global user.name "Automatic Deployment (Travis CI)";
+        git config --global user.email "espp-devel@listserv.mpip-mainz.mpg.de";
+        git add --all && git commit -m "Documentation Update";
+
+    - name: deploy documentation
+      run: |
+        cd html
+        openssl aes-256-cbc -K $encrypted_194b3d1e9306_key -iv $encrypted_194b3d1e9306_iv -in $HOME/docker/espressopp/doc/ug/deploy.enc -out ~/.ssh/id_rsa -d;
```

Comment:
In Github actions we either should add a PAT in a secret to do the deploy or generate a new deploy ssh key and store it private key in a secret.
```
echo ${{ SECRET.private_deploy_key }} | base64 -d > ~/.ssh/id_rsa
```

Reply:
https://github.community/t/github-actions-ci-how-to-use-store-deploy-key-to-download-from-another-private-repo/16113/7

Reply:
done